api/audio_codecs/audio_encoder.cc

```diff
@@ -110,5 +110,19 @@ ANAStats AudioEncoder::GetANAStats() const {
   return ANAStats();
 }
 
+size_t AudioEncoder::AppendPreEncodeData(rtc::ArrayView<const int16_t> audio,
+                                      rtc::Buffer* encoded) {
+  const size_t old_size = encoded->size();
+  for (const int16_t it : audio) {
+    uint8_t arr[2] = {
+       static_cast<uint8_t>((it >> 8) & 0x00ff),
```

Comment:
Is endianess an issue here?

Reply:
I assumed we will have decoder (and this is how I tested), so the most important was to be consistent with decoder.
Otherwise, this is good question. I think I'll convert it to Little Endian (as more popular architecture). Another alternative is to rely on real architecture (by using C like union implementation) or expose this information as part of API, but It may look too complicated, because who knows what output SDP device will decode it.